### DISCUSSION TITLE
codehilite not working post upgrade

### DISCUSSION THREAD

**Discussion by emh1908** [squidfunk/mkdocs-material#3422] [Q&A] [answered]
Hi - I hope someone can help with this - apologies in advance if I've missed something obvious (I'm not a techie). 

I’ve just got a new laptop on which I’ve installed the various components.

![image](https://user-images.githubusercontent.com/36504760/148083216-4f1ae781-e14b-4295-a81c-22ecaa999973.png)

In the yaml file, I have this: 
![image](https://user-images.githubusercontent.com/36504760/148083330-9109d38f-a24d-4c4c-a953-50a222a26089.png)

The codehilite is no longer working.
![image](https://user-images.githubusercontent.com/36504760/148083549-75eb8b93-3c5a-4bc6-9460-f5aca54266d2.png)

Thank you.

**Comment by facelessuser** [comment, marked as answer]
You are using some unsupported extensions. Material does not formally support `markdown-fenced-code-tabs`. It uses [`pymdownx.superfences`](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown-extensions/?h=#superfences) which does not use `codehilite`. Content tabs are provided by [`pymdownx.tabbed`](https://squidfunk.github.io/mkdocs-material/setup/extensions/python-markdown-extensions/?h=#tabbed) extension.

Regardless of whether something incidentally worked before is inconsequential. With that said, you are free to explore using other extensions, but you may have to provide your own CSS styling to make them work.

**Reply by squidfunk** [reply]
An example of how you can achieve the same with the extensions mentioned by @facelessuser is here:
https://squidfunk.github.io/mkdocs-material/reference/content-tabs/#grouping-code-blocks

**Reply by emh1908** [reply]
Thank you for this link. Much appreciated.

**Comment by emh1908** [comment]
@facelessuser Thank you - I've inherited the project so learning as I go. I appreciate your input.
